arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/VectorStylesViewController.swift

```diff
@@ -26,8 +26,8 @@ class VectorStylesViewController: UITableViewController {
     
     weak var delegate: VectorStylesVCDelegate?
     
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
```

Comment:
By setting UI in `viewWillAppear`, it will cause `UITableViewAlertForLayoutOutsideViewHierarchy` warning.

I cannot think of a good way to update a static table before it is loaded, so maybe we'll need Phil's help.

With this change there will be a short delay before the checkmark appears, while it doesn't throw warnings anymore.

Reply:
I haven't tested it, but I think the same technique used by `OptionsTableViewController` could be employed. Simply override `tableView(_:cellForRowAt:)` to set the accessory type:

```swift
override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
    let cell = super.tableView(tableView, cellForRowAt: indexPath)
    cell.accessoryType = itemIDs[indexPath.row] == selectedItemID ? .checkmark : .none
    return cell
}
```

And then reload/deselect rows as appropriate in `tableView(_:didSelectRowAt:)`.

Reply:
I couldn't reload the cells because they came out blank after. I fixed the checkmark delay but it may not be as efficient as reloading the cells. Please let me know what you think!